README.md

```diff
@@ -464,14 +486,11 @@ print_to_hash(**success_data) # [:success, :ok, 1]
 
 ### Railway Oriented Programming
 
-This feature/pattern is also known as ["Railway Oriented Programming"](https://fsharpforfunandprofit.com/rop/).
-
-The idea is to chain blocks and creates a pipeline of operations that can be interrupted by a failure.
-
-In other words, the block will be executed only if the result is a success.
-So, if some block returns a failure, the following blocks will be skipped.
+["Railway Oriented Programming (ROP)"](https://fsharpforfunandprofit.com/rop/)  is a programming paradigm that involves linking blocks together to form a sequence of operations, also known as a pipeline.
```

Comment:
ROP is not a programming paradigm. It is a concept/technique (developed on the functional programming paradigm). So instead of saying “Is programming paradigm” is a technique. Does it make sense to you?